doc/Installation/Installation-CentOS-7-Apache.md

```diff
@@ -171,6 +177,11 @@ Now head to the web installer and follow the on-screen instructions.
 
     http://librenms.example.com/install.php
 
+The web installer will prompt you to create a `config.php` file in your librenms install location. Please remember to set the permissions on config.php after you copied the on-screen contents to the file. Run:
```

Comment:
This is definitely not needed. It is ok for the config.php file to have different permissions, as long as the web server can read it, and it definitely can since it wrote it.

Sure, validate.php will complain about it, but that is minor.

Reply:
Yeah, web server has trouble making that file because of permissions and we can't fix the permissions for that folder because it would mess permissions.
https://github.com/ConsoleTVs/Charts